repos/c2sm/packages/icon/package.py

```diff
@@ -144,12 +148,6 @@ class Icon(AutotoolsPackage, CudaPackage):
     variant('sct', default=False, description='Enable the SCT timer')
     variant('yaxt', default=False, description='Enable the YAXT data exchange')
 
-    claw_values = ('std', 'validate')
-    variant('claw',
-            default='none',
-            values=('none', ) + claw_values,
-            description='Enable CLAW preprocessing')
-
```

Comment:
Since this variant is gone CLAW is not used anymore as a package.
I suggest to remove this package in a separate PR later.

Reply:
I will 👍